### PR TITLE
Jetpack Plan: filter video mime types

### PR DESCRIPTION
**Pull request by retrofox**
This PR filters the video mime-types considering: the site type (Atomic or Jetpack), the `videopress` feature, and of course, it's also subject to the `videopress` module activation.

#### Changes proposed in this Pull Request:

*

#### Testing instructions:

* Go to '..'
*

### CODE REVIEW

File: projects/plugins/jetpack/class.jetpack-plan.php [hunk 1]
```diff
@@ -344,3 +344,44 @@ public static function supports( $feature ) {
...
+	};
+
+	// `videopress` feature is supported. Bail early.
+	if ( Jetpack_Plan::supports( 'videopress' ) ) {
+		return $mimes;
+	};
+
+
+	// Remove `video/` mimes from the list.
+	return array_filter(
```
Comment (pablinos) [inline projects/plugins/jetpack/class.jetpack-plan.php:373]: I'm not sure we want to remove the ability to upload videos. If VideoPress is not available/active then people should still be able to upload videos to their site.

Comment (retrofox) [reply projects/plugins/jetpack/class.jetpack-plan.php:373]: agree, something (wrongly) handled in the block above. When the module is not active, bail early.

Comment (retrofox) [reply projects/plugins/jetpack/class.jetpack-plan.php:373]: fixed here 05912df5b6cd48a2e7b41e76fb675452bc19a93c

Comment (pablinos) [reply projects/plugins/jetpack/class.jetpack-plan.php:373]: Under what circumstances would we get here? If the module is active then the plan must support it, and if it's inactive then we bail early.

I don't think this class is available on WPCOM, and it's only there we want to remove these mime types. It's possible I'm just confused. There are a lot of moving parts to this.

**Pull request closed without merging**